content/ja/docs/contribute/review/reviewing-prs.md

```diff
@@ -47,12 +78,17 @@ weight: 10
     - PRの説明を読み、行われた変更について理解し、関連するissueがあればそれも読みます。
     - 他のレビュアのコメントがあれば読みます。
     - **Files changed**タブをクリックし、変更されたファイルと行を確認します。
-    - **Conversation**タブの下にあるPRのbuild checkセクションまでスクロールし、**deploy/netlify**の行の**Details**リンクをクリックして、Netlifyのプレビュービルドで変更点をプレビューします。
+    - **Conversation**タブの下にあるPRのbuild checkセクションまでスクロールし、Netlifyのプレビュービルドで変更点をプレビューします。これはスクリーンショットです(これは、GitHubのデスクトップサイトを見せています; タブレットやスマートフォンデバイスでレビューしている場合は、GitHubウェブのUIは少し異なります):
```

Comment:
```suggestion
    - **Conversation**タブの下にあるPRのbuild checkセクションまでスクロールし、Netlifyのプレビュービルドで変更点をプレビューします。これはスクリーンショットです(これは、GitHubのデスクトップサイトを見せています。 タブレットやスマートフォンデバイスでレビューしている場合は、GitHubウェブのUIは少し異なります):
```
`;` in English usually means `。` in Japanese I think.

Reply:
Fixed.